src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -81,6 +81,11 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
       verticalAlign: 'middle',
       cursor: 'pointer',
 
+      // @Bugaa92: temporary fix until semantic-ui-css is not referenced anymore
+      ':-moz-focusring': {
+        outline: 0,
```

Comment:
is the same one currently used in Teams to achieve the same effect?

Reply:
yes, checked